### PR TITLE
stubby: bump to 0.4.2

### PR DESCRIPTION
**Pull request by rmandrad**
Maintainer: me / @neheb  (find it by checking history of the package Makefile)
Compile tested: aarch64/ipq8074
Run tested: aarch64/ipq8074

Description:  upgrade stubby to latest 0.4.2

Signed-off-by: Rudy Andram <rmandrad@gmail.com>

### CODE REVIEW

File: net/stubby/Makefile [hunk 1]
```diff
@@ -5,8 +5,8 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=stubby
-PKG_VERSION:=0.4.0
-PKG_RELEASE:=6
```
Comment (neheb) [inline net/stubby/Makefile:9]: reset to 1

**Comment by neheb** [comment]
```
Package HASH check failed
```

File: net/stubby/Makefile (hunk 1 above)
Comment (rmandrad) [reply net/stubby/Makefile:9]: done

**Comment by 1715173329** [comment]
Commit title should be `stubby: bump to 0.4.2`
And please commit it with a proper name and SoB, see https://openwrt.org/submitting-patches

**Comment by rmandrad** [comment]
what is holding commiting this !

**Pull request closed without merging**